### DISCUSSION TITLE
Path service not found

### DISCUSSION THREAD

**Discussion by M1Cody** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#309] [Q&A] [answered]
Im using this integeration [https://github.com/KrzysztofHajdamowicz/home-assistant-vacuum-styj02ym](url) and it works fine for zone and point cleanup but not for path follow 
![image](https://user-images.githubusercontent.com/29939651/148930019-6704123f-9f4e-46c1-95f9-5c0571909b4d.png)

**Comment by PiotrMachowski** [comment, marked as answer]
Have you added the script that is linked in [documentation](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card#supported-templates)?
